borsh-rs/borsh/src/de/mod.rs

```diff
@@ -273,6 +284,17 @@ where
     }
 }
 
+impl<T> BorshDeserialize for Cow<'_, [T]>
```

Comment:
Is it possible to implement it for `Cow<'_, T>`?

Reply:
I haven't tried, but I think we cannot at the moment since it will collide with `str` and `[T]` implementations without specialization. We have experience working around this for Vec<u8>, but I don't want to dive into that at this moment.

Reply:
Can we only implement it for `Cow<'_, T>`?
It will cover `String`, but not sure if this is enough for you

Reply:
I actually implemented generic Cow<T> and there is no need in specialization!